src/Analyzers/Core/CodeFixes/ConflictMarkerResolution/AbstractConflictMarkerCodeFixProvider.cs

```diff
@@ -298,11 +299,14 @@ private static bool IsConflictMarker(TextLine currentLine, char ch)
 
             static CodeAction CreateCodeAction(string title, Func<CancellationToken, Task<Document>> action, string equivalenceKey)
             {
-#if CODE_STYLE
-                return CodeAction.Create(title, action, equivalenceKey);
-#else
-                return CodeAction.DocumentChangeAction.Create(title, action, equivalenceKey, CodeActionPriority.High);
+                var codeAction = CodeAction.Create(title, action, equivalenceKey, CodeActionPriority.High);
+
+#if !CODE_STYLE
+                // Backdoor that allows this provider to use the high-priority bucket.
+                codeAction.CustomTags = codeAction.CustomTags.Add(CodeAction.CanBeHighPriorityTag);
 #endif
```

Comment:
however, because it is high pri, the feature has to use this backdoor for that to be ok.  it will be clamped otherwise.